src/components/dropdown/dropdown.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import {
+  ListboxInput,
+  ListboxButton,
+  ListboxPopover,
+  ListboxList,
+  ListboxOption,
+} from '@reach/listbox';
+import '@reach/listbox/styles.css';
+import styled from 'styled-components';
+
+const StyledListboxInput = styled(ListboxInput)`
+  cursor: pointer;
```

Comment:
So there is no pointer by default. Maybe that's okay? Can you compare with the native select? Thanks.

Reply:
The native select has no pointer. I removed it from the dropdown component